src/SimpleGraphs/generators/staticgraphs.jl

```diff
@@ -47,6 +47,61 @@ function CompleteBipartiteGraph(n1::T, n2::T) where {T <: Integer}
     return SimpleGraph(ne, fadjlist)
 end
 
+"""
+    CompleteMultipartiteGraph(partitions)
+
+Create an undirected [complete bipartite graph](https://en.wikipedia.org/wiki/Complete_bipartite_graph)
+with `sum(partitions)` vertices and `prod(partitions)` edges.
+"""
+function CompleteMultipartiteGraph(partitions::AbstractVector{T}) where {T <: Integer}
+    length(partitions) == 1 && return SimpleGraph{T}(partitions[1])
+    length(partitions) == 2 && return CompleteBipartiteGraph(partitions[1], partitions[2])
+    (length(partitions) == 0 || any(x -> x <= 0, partitions)) && return SimpleGraph(T)
```

Comment:
I just saw that for `CompleteBipartiteGraph(2, 0)` you get a graph with 2 vertices and no edges, so in the interest of consistency, `CompleteMultipartiteGraph([2,0])` should probably do the same.

Reply:
So for this one, I just implemented it how the standard definition is and I hoped a discussion would happen on what to do if one partition size is 0. Currently, CompleteMultipartiteGraph([2,0]) just calls CompleteBipartiteGraph(2,0) since the check for the number of partitions comes before the check if any partitions are  <= 0.

Reply:
I think the best way is to just skip over the partitions of size <= 0. That is kind of what you would expect - if you add a partition of size <= 0 to MultipartiteGraph, then just nothing new gets added. It might be enough for you to just remove the the security check for such partitions, then your code might already work as expected (but double check that)